modules/btcpay/btcpay.php

```diff
@@ -334,18 +334,29 @@ public function hookDisplayOrderDetail(array $params): ?string
 		$storeID = $this->configuration->get(Constants::CONFIGURATION_BTCPAY_STORE_ID);
 
 		try {
+			// Get the payment methods
+			$paymentMethods = $client->invoice()->getPaymentMethods($storeID, $invoiceId);
+
+			// Determine if any payments have been made via BTCPay Sever
+			$hasPayments = array_reduce($paymentMethods, static fn ($hasPayments, $paymentMethod) => $hasPayments || !empty($paymentMethod->getPayments()), false);
+
+			// If not, there is no need to show anything
```

Comment:
Not sure where this exactly displays but I assume on the order overview. Wouldn't it be helpful to see that invoice id even if no payment has been made yet? Or is it useless in that context?

Reply:
It is indeed the order overview on the customers side. I thought about including the invoice itself, but if the customer didn't make a payment via BTCPay Server (but instead via cash/PayPal/etc), it doesn't really make sense to show it. At least, that was my train of thought.

Reply:
According to https://docs.cloud.prestashop.com/10-validation-checklist/ I should check if the order was created via our payment module. If it is, we will just show it (no matter if it was paid via BTCPay Server or not). 

An upcoming push will revert this change and change it to be more inline with what Prestashop demands.